library/spdm_responder_lib/libspdm_rsp_set_certificate.c

```diff
@@ -245,7 +260,8 @@ libspdm_return_t libspdm_get_response_set_certificate(libspdm_context_t *spdm_co
 
 #if LIBSPDM_CERT_PARSE_SUPPORT
         /*check the cert_chain*/
-        result = libspdm_set_cert_verify_certchain(cert_chain, cert_chain_size,
+        result = libspdm_set_cert_verify_certchain(spdm_version,
+                                                   cert_chain, cert_chain_size,
                                                    spdm_context->connection_info.algorithm.base_asym_algo,
                                                    spdm_context->connection_info.algorithm.base_hash_algo,
                                                    is_device_cert_model);
```

Comment:
The last parameter is changed to `uint8_t cert_model`.

Reply:
Wow. Thank you for type checking when the C compiler could not. 😆